docker-compose.yml

```diff
@@ -18,3 +18,32 @@ services:
     # Update this path to match your directory structure
     volumes:
       - ./lib/deqm_test_kit/igs:/home/igs
+  fhir_validator_app:
```

Comment:
If it's not useful to have the standalone validator app, you can remove this. If it is useful, you can find it at `http://localhost/validator`.